### DISCUSSION TITLE
索引功能跟ChatPDF网站是不是同样的效果

### DISCUSSION THREAD

**Discussion by OkamiFeng** [GaiZhenbiao/ChuanhuChatGPT#418] [General]
还是说不如ChatPDF的实际效果的，有考虑整合ChatPDF进来吗

**Comment by GaiZhenbiao** [comment]
差不多，可以自行体验

**Reply by Nativu5** [reply]
如果是读 PDF 的话，个人推荐 https://github.com/MrPeterJin/researchgpt ，这个效果更好，而且消耗的 Token 少。
